app/views/locations/index.html.erb

```diff
@@ -11,57 +11,102 @@
 
 <%= render partial: "organisations/headings", locals: { main: @scheme.service_name, sub: nil } %>
 
-<%= render SubNavigationComponent.new(items: scheme_items(request.path, @scheme.id, "Locations")) %>
+<% if FeatureToggle.new_locations_table_layout_enabled? %>
```

Comment:
I would move the feature toggle just above the table ( `<%= govuk_table do |table| %>` ) and only conditionally display the table content instead of the whole page

Reply:
A bit less repetition methinks

Reply:
Yup, that makes sense!